src/SWP/Bundle/TemplateEngineBundle/Command/ThemeGenerateCommand.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Fixtures Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\TemplateEngineBundle\Command;
+
+use SWP\Bundle\CoreBundle\Document\Tenant;
+use SWP\Component\MultiTenancy\Exception\OrganizationNotFoundException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Filesystem\Filesystem;
+
+class ThemeGenerateCommand extends ContainerAwareCommand
+{
+    const THEMES_DIR = 'themes';
+    const HOME_TWIG = 'home.html.twig';
+    const THEME_CONFIG_JSON = 'theme.json';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('theme:generate')
+            ->setDescription('Creates basic theme structure with routes and empty templates.')
+            ->addArgument(
+                'organizationName',
+                InputArgument::REQUIRED,
+                'Organization Name',
+                null
+            )
+            ->addArgument(
+                'themeName',
+                InputArgument::REQUIRED,
+                'Theme Name',
+                null
+            )
+            ->setHelp(
+                'The <info>%command.name%</info> command creates a skeleton theme in your application themes folder (app/themes)'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $tenant = $this->getTenant($input, $output);
+            if (null === $tenant) {
+                return;
+            }
+
+            $themeName = $input->getArgument('themeName');
+            $themeDir = implode(\DIRECTORY_SEPARATOR, [$this->getContainer()->get('kernel')->getRootDir(), self::THEMES_DIR, $tenant->getCode(), $themeName]);
+
+            $fileSystem = new Filesystem();
+            if ($fileSystem->exists($themeDir)) {
+                $output->writeln('Theme '.$themeName.' already exists!');
+
+                return;
+            }
+
+            $themeDir = $this->createSkeleton(new Filesystem(), $tenant->getCode(), $themeName);
```

Comment:
How about different name for this variable? As this same one is used few lines before and you override it here. If it's this same as before (result of createSkeleton and original $themeDir) then just don't assign it again to variable.